uds/uds_communications/TransportProtocols/Can/CanTp.py

```diff
@@ -279,7 +286,7 @@ def send(self, payload, functionalReq=False):
             if(timeoutTimer.isExpired()):
                 raise Exception("Timeout waiting for message")
 
-            sleep(0.001)
+            #sleep(0.01)
 
```

Comment:
Why has this been changed? Is it important? The sleep was in to mitigate intermittent issues with CanTp correctly processing the responses